connectivity/check/test.go

```diff
@@ -178,7 +191,7 @@ func (t *Test) Run(ctx context.Context) error {
 
 		sn := t.scenarioName(s)
```

Comment:
Nit: probably can move this inside the `t.Logf` below now that it's the only user of `sn`.